helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
 class DrydockConfig(object):
```

Comment:
Makes sense.  Oslo config files (or INI -- whatever ant to call them) are preferred, that way when we create a drydock helm chart, we can build its configuration at install time.